core/unit_test/tstNeighborList.hpp

```diff
@@ -263,37 +263,49 @@ TEST( TEST_CATEGORY, linked_cell_stencil_test ) { testLinkedCellStencil(); }
 //---------------------------------------------------------------------------//
 TEST( TEST_CATEGORY, verlet_list_full_test )
 {
+#ifndef KOKKOS_ENABLE_OPENMPTARGET // FIXME_OPENMPTARGET
     testVerletListFull<Cabana::VerletLayoutCSR, Cabana::TeamOpTag>();
+#endif
     testVerletListFull<Cabana::VerletLayout2D, Cabana::TeamOpTag>();
 
-#ifndef KOKKOS_ENABLE_HIP // FIXME_HIP
+#ifndef KOKKOS_ENABLE_HIP          // FIXME_HIP
```

Comment:
We should not address it here but I did not realize this was still disabled.

Reply:
I forgot about it. It's removed in #312 though